testtools/mock_storage.go

```diff
@@ -5,31 +5,22 @@ import (
 	"sync"
 )
 
-type MockStorage struct {
+type InMemoryStorage struct {
```

Comment:
Does in-memory store actually store anything?

Reply:
Yes